### DISCUSSION TITLE
Open registration to default group (Simple Sign-up)

### DISCUSSION THREAD

**Discussion by epheterson** [mealie-recipes/mealie#1445] [Feature Request]
**Before submitting this feature request I have**
- [ X ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [ X ] Reviewed the existing feature requests and my feature has not been requested.
- [ X ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
There's an option to set an `ALLOW_SIGNUP` docker environment variable, though the flow is confusing for a user to figure out. The user must have a group token, know to click "Join Group" and use that token vs. making a new group. It gives too many options to the casual user.

Current enrolling of new members is a cumbersome process of admin generating an invite link, sharing that link, communicating the link is one-time use, sending a second link for where they should login after / visit, and hoping they save the right one. It'd be great to send only the base URL and have them Register and re-visit with the same URL.

**(Optional): Suggest A Solution**
Registration would be for a default group (e.g. Home or Guests, set my admin). There would be no option to make a new group, and no requirement for any other data. Essentially the registration page would behave how an invite link works now. 

The user clicks Register, enters username/email/password and signs in, easy peasy.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by wuast94** [comment]
this would be a very nice change, i want to make mine public so my friends can share and register my instance. but this token part is quite confusing, also i cant even figure out whats the token needed or where to generate it

**Comment by chris5a** [comment]
I agree with this feature request. New user registration needs to be simplified.

When looking at the Mealie site features it says for users "Add new users with sign-up links or simply create a new user in the admin panel." I don't see any "sign-up links".

For user creation users can self register as suggested above and added to a default group or put in a pending state waiting for approval by an administrator. Or an admin can create a new user by entering a new user email address and group and it will email a one time link to that email for that new user to complete registration by putting in their full name and password.

Currently it is way too complicated for both new users trying to register and admins trying to send all sorts of group IDs or links.

Thanks

**Comment by TheInfamousAlk** [comment]
I agree with this too, is there any traction on this? I want to open my instance to a group of 5-15 friends, and it will be a nightmare to manage all of them, and their families and friends. Just inviting them will be annoying, then they'll have to contact me to get a link whenever they want to add someone, and that's assuming they don't get confused with the group/household system.
